cli_tools/daisy/main.go

```diff
@@ -282,3 +285,33 @@ func main() {
 		}
 	}
 }
+
+func printPerfProfile(workflow *daisy.Workflow) {
+	if !*printPerf {
+		return
+	}
+
+	timeRecords := workflow.GetStepTimeRecords()
+	if len(timeRecords) == 0 {
+		return
+	}
+
+	wfStartTime := time.Now()
+	wfEndTime, _ := time.Parse(time.RFC3339, "0001-01-01T00:00:00+00:00")
+	fmt.Println("\nPerf Profile:")
+	for _, r := range timeRecords {
+		if wfStartTime.After(r.StartTime) {
+			wfStartTime = r.StartTime
+		}
+		if wfEndTime.Before(r.EndTime) {
+			wfEndTime = r.EndTime
+		}
+		fmt.Printf("- %v: %v\n", r.Name, formatDuration(r.EndTime.Sub(r.StartTime)))
+	}
+	fmt.Printf("Total time: %v\n\n", formatDuration(wfEndTime.Sub(wfStartTime)))
+}
+
+func formatDuration(d time.Duration) string {
+	s := int(d.Seconds())
+	return fmt.Sprintf("%v:%v:%v", s/3600, s/60%60, s%60)
```

Comment:
Should add units, e.g. %vh:%vm:%vs

Reply:
0:10:32 is a nice format as well. I can directly copy it to sheet or excel for calculation. 0h:10m:32s is not accepted by those sheets program.

Reply:
The problem here is it's not possible to know what the values mean without looking at the code. At least add units before/after the value so it's obvious to anyone reading the perf report

Reply:
Sure. I though people have aligned understanding on that format. Changed.